recipe/meta.yaml

```diff
@@ -19,13 +19,15 @@ requirements:
     - cmake
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - ninja
+    - ninja-base
   host:
-    - aws-c-common 0.6.8
-    - aws-checksums 0.1.11
+    - aws-c-common 0.8.5
+    - aws-checksums 0.1.13
+    - aws-c-io 0.13.10
   run:
-    - aws-c-common >=0.6.8,<0.6.9.0a0
-    - aws-checksums >=0.1.11,<0.1.12.0a0
+    - aws-c-common # pinning by run_exports
```

Comment:
I guess you're putting these here to be explicit? It's not really usual in the recipes, we'd just leave it to the run_exports

Reply:
suggestion from @JeanChristopheMorinPerso

Reply:
Yep, it's preferable to be explicit than implicit.